docker/install_docker.sh

```diff
@@ -146,16 +146,41 @@ else
 	DOCKER_COMPOSE_VERSION="1.25.5"
 
 	echo "Installing Docker-Compose v${DOCKER_COMPOSE_VERSION}..."
-	#if [ "$(uname -m)" = "x86_64" ]; then
-	#	$SUDO_E curl --silent -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` -o /usr/bin/docker-compose
-	#	$SUDO chmod +x /usr/bin/docker-compose
-	#else
 
-	#github doesn't have aarch64 binary releases for docker-compose, install via pip3 instead (on every architecture, to simplify this script)
-	$SUDO pip3 install --upgrade pip
-	$SUDO pip3 install docker-compose==${DOCKER_COMPOSE_VERSION}
+	# Check if the latest version of pip is supported by the version of python installed
+	# In particular, Ubuntu 16's version of python (v3.5) does not support the latest verison of pip
+	MIN_PYTHON_VERSION_MAJOR=3
+	MIN_PYTHON_VERSION_MINOR=6
+	PYTHON_VERSION_TEST="
+import sys
+if  (sys.version_info.major > $MIN_PYTHON_VERSION_MAJOR or 
+	(sys.version_info.major == $MIN_PYTHON_VERSION_MAJOR and sys.version_info.minor >= $MIN_PYTHON_VERSION_MINOR)): 
+	sys.exit(0)
+sys.exit(1)
+"
+	if python3 -c "$PYTHON_VERSION_TEST"; then 
+		# prefer to install with pip3 if possible since github doesn't have aarch64 binary releases for docker-compose
 
-	#fi
+		# pip3 recommends -H when running with sudo to prevent creating root owned files in the user's home dir
+		PIP3_CMD="pip3"
```

Comment:
Is there a reason why we're invoking `pip3` directly instead of using `python3 -m pip ...`? 
The [pip documentation](https://pip.pypa.io/en/stable/user_guide/#) recommends invoking pip with python. 
I came across the same warning shown here when troubleshooting the 16.04 issue:
![image](https://user-images.githubusercontent.com/24681559/160883877-037b66b5-9eb3-4db9-8aee-1842738bbe07.png)
[Documentation](https://pip.pypa.io/en/stable/installation/#upgrading-pip) also shows upgrading pip via python. If future pip versions actually do drop support for invoking pip with pip, it could cause issues for newer distros/the future.

Reply:
Huh, its just the way I've always ran pip. I'll get the command changed int the PR to use `python3 -m pip`